modules/rostests/rosautotest/CWineTest.cpp

```diff
@@ -64,11 +64,27 @@ CWineTest::GetNextFile()
     WIN32_FIND_DATAW fd;
 
     /* Did we already begin searching for files? */
-    if(m_hFind)
+    if (m_hFind)
     {
         /* Then get the next file (if any) */
-        if(FindNextFileW(m_hFind, &fd))
-            FoundFile = true;
+        if (FindNextFileW(m_hFind, &fd))
+        {
+            // printf("cFileName is '%S'.\n", fd.cFileName);
+            /* if it was NOT rosautotest.exe then proceed as normal */
+            if (wcsicmp(fd.cFileName, TestName) != 0)
+            {
+                FoundFile = true;
+            }
+            else
+            {
+                /* It was rosautotest.exe so get the next file (if any) */
+                if (FindNextFileW(m_hFind, &fd))
+                {
+                    FoundFile = true;
+                }
+                // printf("cFileName is '%S'.\n", fd.cFileName);
+            }
```

Comment:
It might make sense to share this code with the diff below (move it into a function or so).
That way if the ignore-list ever gets bigger than 1 item it will be a trivial change in one place.

Reply:
I prefer to leave this for a future improvement for now. Thanks.